tests/orm/utils/test_serialize.py

```diff
@@ -179,3 +179,12 @@ def test_dataclass():
 
     deserialized = serialize.deserialize_unsafe(serialized)
     assert deserialized == obj
+
+
+def test_serialize_link():
+    """Test you can serialize and deserialize a link"""
+    from aiida.orm.utils.managers import NodeLinksManager
+    node = orm.Data().store()
+    link = NodeLinksManager(node=node, link_type=LinkType.CREATE, incoming=False)
+    deserialized = serialize.deserialize_unsafe(serialize.serialize(link))
```

Comment:
I would assert `isinstance(deserialized, NodeLinksManager)` here and asser that the `node`, `link_type` and `incoming` attributes are as they were before

Reply:
resolved in commit [d65362f](https://github.com/aiidateam/aiida-core/pull/6199/commits/d65362fcfc062458a08df51b4cb8541e25e2aa15)